import_export/resources.py

```diff
@@ -263,6 +264,10 @@ def __new__(cls, name, bases, attrs):
                     for option in dir(options)
                     if not option.startswith("_") and hasattr(options, option)
                 ]:
+                    if option == "model" and isinstance(getattr(options, option), str):
```

Comment:
I notice that `getattr(options, option)` is called multiple times.  Would it make sense to call once and assign to a var, e.g.

```
opt_val = getattr(options, option)
```

Reply:
Done